sidestep_engine/data/preprocess.py

```
@@ -460,6 +464,8 @@ def _pass1_light(
 
             except Exception as exc:
                 failed += 1
+                print(exc)
+                import pdb; pdb.set_trace()
```

Ruff (0.15.7)

[error] Import for `pdb` found

(T100)